stdlib/experimental/http/http_experimental.go

```diff
@@ -13,22 +14,27 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"time"
 )
 
 // maxResponseBody is the maximum response body we will read before just discarding
 // the rest. This allows sockets to be reused.
 const maxResponseBody = 512 * 1024 // 512 KB
 
+// http.NewDefaultClient() does default to 30
+const defaultTimeout = int64(30)
+
 // http get mirrors the http post originally completed for alerts & notifications
 var get = values.NewFunction(
 	"get",
 	semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
 		Parameters: map[string]semantic.PolyType{
 			"url":     semantic.String,
 			"headers": semantic.Tvar(1),
+			"timeout": semantic.Int,
```

Comment:
Can you use a duration type instead of an integer? That way user can specify `4s` or `500ms` in which ever units works best for their use case.

Reply:
Done!